myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -71,7 +80,7 @@ The above formula for computing weights is a very nice and simple approach, but
 
 ## Stacking
 
-The third approach implemented in PyMC3 is know as _stacking of predictive distributions_ and it has been recently [proposed](https://arxiv.org/abs/1704.02030). We want to combine several models in a metamodel in order to minimize the diverge between the meta-model and the _true_ generating model, when using a logarithmic scoring rule this is equivalently to:
+The third approach implemented in PyMC is known as [_stacking of predictive distributions_](https://arxiv.org/abs/1704.02030). We want to combine several models in a metamodel in order to minimize the divergence between the meta-model and the _true_ generating model, when using a logarithmic scoring rule this is equivalent to:
```

Comment:
same here (it's actually the same paper as above so so far 1 reference to add to bibtex)